llpc/builder/llpcTargetInfo.h

```diff
@@ -38,6 +38,15 @@ namespace Llpc
 
 using namespace llvm;
 
+// Represents graphics IP version info. See https://llvm.org/docs/AMDGPUUsage.html#processors for more
+// details.
+struct GfxIpVersion
+{
+    uint32_t        major;              ///< Major version
```

Comment:
I think you don't need the "///<" comments (just use "//") within LLPC/LGC. Other commits have the similar problem.